management/db.py

```diff
@@ -448,10 +448,39 @@ def delete_freezer(user_id,victim_id):
     conn.commit()
     return True
 
-def add_standoff():
-    """Add a new kill condition to the database."""
-    pass
-    # TODO
+def add_standoff(victim_id,role,murderer):
+    # NOT TESTED YET - PLEASE DO
```

Comment:
need to test before merge

Reply:
Whoops! Forgot to remove that comment. It has been tested, and the evil dr. Travis is currently testing it too.